stubs/docutils/docutils/frontend.pyi

```diff
@@ -60,6 +62,14 @@ class OptionParser(optparse.OptionParser, SettingsSpec):
     default_error_encoding_error_handler: ClassVar[str]
     config_section: ClassVar[str]
     version_template: ClassVar[str]
+    def __init__(
+        self,
+        components: Iterable[Type[Parser]] = ...,
+        defaults: Mapping[str, Any] = ...,
```

Comment:
This can be `None`, and is `None` by default:

```suggestion
        defaults: Mapping[str, Any] | None = ...,
```

Reply:
Thanks! Fixed. Interestingly this finally flagged the `docutils.frontend.OptionParser.__init__` as an unused allowlist entry which I have now removed.
I'm pretty confident we've got all the optional arguments now as the `Directive.__init__` has none at all.